src/book/01-team/02-joining/03-induction.mdx

```diff
@@ -6,14 +6,34 @@ description: The NUbots lab induction process for new members.
+
+The CESE Work, Health and Safety Canvas site contains all the information needed to obtain swipe-card access to the NUbots lab. Ask the NUbots Team Leader for a link to self-enrol in the site if you are not already enrolled. On the site homepage, scroll to the bottom of the page and click on the 'Newcastle Robotics Laboratory' banner.
+
+## Safety Documentation
+
+The [Electrical Engineering Safety Manual](https://www-eng.newcastle.edu.au/eecs/ect/oh&s/) is required reading, and will help you with the EE/ES Lab Induction that you must complete in the next step. The second required reading is the [risk assessment](./assets/risk-assessment.pdf) for the lab.
+
+## EE/ES Lab Induction
+
+After reading the safety documentation, you are required to complete the EE/ES Lab Induction. This can be accessed by clicking the link in step 2 on Canvas. Once you have finished the quiz, navigate back to the NUbots induction page - if you instead click 'next' in Canvas, it will take you to further quizzes for Electrical Engineering, which you are **not** required to complete.
+
+You must score **100%** in the quiz. The [Electrical Engineering Safety Manual](https://www-eng.newcastle.edu.au/eecs/ect/oh&s/) accompanies the quiz. **This quiz must be completed by all team members each year**, even if you already have swipe card access.
+
+Students working in the EE labs for their coursework will have already completed this quiz. If you have already completed the quiz in this calender year, you do not need to do it again.
 
 ## The NUbots Lab Induction
 
-The second part of the process is the NUbots lab induction. This is an in-person induction done with a Team Leader or Team Safety Officer in the lab. This induction walks new members though the available safety and emergency equipment in the lab, how to safely handle batteries and robots, and what to do when there is an emergency.
+The next part of the process is the NUbots lab induction. This is an in-person induction done with a Team Leader or Team Safety Officer in the lab. This induction walks new members though the available safety and emergency equipment in the lab, how to safely handle batteries and robots, and what to do when there is an emergency.
+
+Upon completion of the NUbots lab induction, new members must must fill out the [lab induction checklist](./assets/lab-induction-checklist.pdf). They are to obtain a digital copy to upload to Canvas, and the hardcopy will be kept in the lab as a record. A copy of the checklist will be provided to the student at the beginning of the induction.
+
+## Request Swipe Card Access
+
+The final step on Canvas involves an embedded app that allows you to request access. After signing in to the app with your student details, you will see a form. Select the NUbots access group and choose Alex Mendes to approve the request. If any steps are not complete, the app will indicate this. Once the steps are all complete, you can submit your request for swipe-card access.
```

Comment:
Just checking that it says Alex here and not Alexandre?

Reply:
Just checked, it's Alexandre in the form so will change it to that

```suggestion
The final step on Canvas involves an embedded app that allows you to request access. After signing in to the app with your student details, you will see a form. Select the NUbots access group and choose Alexandre Mendes to approve the request. If any steps are not complete, the app will indicate this. Once the steps are all complete, you can submit your request for swipe-card access.
```